### PR TITLE
cinnamon: update to 4.8.2

### PR DESCRIPTION
**Pull request by Cyanoxygen**
Topic Description
-----------------

Update the Cinnamon desktop environment to 4.8.2. ~~And, Core 8 is out.~~

Package(s) Affected
-------------------

`xapps` - 2.0.0
`groups/cinnamon` - 4.8.2

Security Update?
----------------

No

Build Order
-----------
 
1. `xapps`
2. `groups/cinnamon`

Architectural Progress
----------------------

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`

Secondary Architectural Progress
--------------------------------

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`

----

After the pull request is merged, all package(s) affected must be rebuilt against the `stable` Git tree and environment (only `stable` repository should be enabled in `sources.list`). This section marks the progress above.

Please, make sure the list of architectures below matches the ones above.

Post-Merge Architectural Progress
---------------------------------

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`

Post-Merge Secondary Architectural Progress
-------------------------------------------

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`

### CODE REVIEW

**Comment by Cyanoxygen** [comment]
## Testing
After a few hours of [debugging](https://github.com/linuxmint/cinnamon/issues/9738), the new desktop finally runs and smoothly.

Here's a screenshot showing Cinnamon 4.8.0 and nemo running in my laptop:

image

## Conclusion

Waiting for the results from other platforms.

File: extra-cinnamon/cinnamon/autobuild/patches/0002-default-theme.patch [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+@@ -645,7 +645,7 @@
+   <schema id="org.cinnamon.theme" path="/org/cinnamon/theme/"
+         gettext-domain="@GETTEXT_PACKAGE@">
+     <key name="name" type="s">
+-      <default>""</default>
++      <default>"cinnamon"</default>
+       <summary>Theme name</summary>
+       <description>
+        The name of the theme
+
```
Comment (MingcongBai) [inline extra-cinnamon/cinnamon/autobuild/patches/0002-default-theme.patch]: What's with all the trailing blank lines?

File: extra-cinnamon/cinnamon-menus/spec [hunk 2]
```diff
@@ -1,3 +1,4 @@
-VER=4.6.1
-SRCTBL="https://github.com/linuxmint/cinnamon-menus/archive/$VER.tar.gz"
-CHKSUM="sha256::534cf3e15fed812e1affb544811e10823edbc56360b52a80ffbabc006c467dc6"
+VER=4.8.1
+SRCS="tbl::https://github.com/linuxmint/cinnamon-menus/archive/$VER.tar.gz"
+CHKSUMS="sha256::a6070591b1e9eaed8b9a332e7e2ecb9fcd90de5565f7e4dc97cd7aeb89a6cc19"
+
```
Comment (MingcongBai) [inline extra-cinnamon/cinnamon-menus/spec]: Trailing blank line here.

File: extra-x11/xapps/spec [hunk 3]
```diff
@@ -1,4 +1,4 @@
-VER=1.8.10
-REL=1
-SRCTBL="https://github.com/linuxmint/xapps/archive/$VER.tar.gz"
-CHKSUM="sha256::a0d542de6dc4d6a318bdf1fad88447db270ab2dd0582f0c6f28008cc77337672"
+VER=2.0.0
+SRCS="tbl::https://github.com/linuxmint/xapps/archive/$VER.tar.gz"
+CHKSUMS="sha256::cc33dea39cc1092e77280ba290b94adf991125d922b285dc20c5a62cc41dac29"
+
```
Comment (MingcongBai) [inline extra-x11/xapps/spec]: Trailing blank line here.

File: extra-cinnamon/nemo/spec [hunk 4]
```diff
@@ -1,4 +1,4 @@
-VER=4.6.5
-SRCTBL="https://github.com/linuxmint/nemo/archive/$VER.tar.gz"
-CHKSUM="sha256::35f18063c4e9e0f94997454482553f48296eb97afe13bcbd56e99a87ea8eb78e"
+VER=4.8.0
+SRCS="tbl::https://github.com/linuxmint/nemo/archive/$VER.tar.gz"
+CHKSUMS="sha256::47fec60b3221f2243df1036bb297fbd602947ee3873d05c768e805246a634f52"
 SUBDIR="nemo-$VER"
```
Comment (MingcongBai) [inline extra-cinnamon/nemo/spec]: I think you can drop all the `SUBDIR=` here. This is a relic from a broken ACBS revision.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Comment by Cyanoxygen** [comment]
Converting to draft as there are some possible bugs on upstream side.

**Comment by MingcongBai** [comment]
@Cyanoxygen Upstream report: https://github.com/linuxmint/nemo/issues/2579

**Comment by Cyanoxygen** [comment]
Reopen as a critical bug was fixed in nemo 4.8.2.
Rebuild and retest required.

**Comment by Cyanoxygen** [comment]
## Testingh

### Platform
- AOSC OS: 8.0
- Arch: `arm64`
- Platform: Raspberry Pi 4
- Xorg: xrdp

Here's a screenshot shwoing Nemo and Cinnamon running on my Raspberry Pi 4.
image

**Review by MingcongBai — APPROVED** [review]

**Comment by Cyanoxygen** [comment]
## Testing
### Platform
- AOSC OS: 8.0
- Arch: amd64
- Xorg: X11

Here's a screenshot showing Cinnamon 4..8.2 running on my Laptop:
image

## Merging eligibility 
Waiting for secondary architectures.

**Comment by Cyanoxygen** [comment]
Successfully built on all necessary platforms.

## Merging eligibility
Yes

**Comment by MingcongBai** [comment]
Package uploads pending. @Cyanoxygen

**Pull request merged**